CSSBot/Services/TheSpookening/spooky.json

```
@@ -79,15 +79,23 @@
     "💡 moth 💡",
     "slaps roof of {0}",
     "cssbot's codebase",
-	"Auto Resolves PR Comments",
-	"Triggered Discord Mod",
-	"Shadow {0}",
-	"Elon Musk's Musk",
-	"CSS {0}",
-	"🎃{0} o'Lantern🎃",
-	"UwU What's this??",
-	"P1 Issue at 12 A.M.",
-	"git push origin --delete master"
+    "Auto Resolves PR Comments",
+    "Triggered Discord Mod",
+    "Shadow {0}",
+    "Elon Musk's Musk",
+    "CSS {0}",
+    "🎃{0} o'Lantern🎃",
+    "UwU What's this??",
+    "P1 Issue at 12 A.M.",
+    "git push origin --delete master",
+    "Anish",
+    "pooby's pms",
```

```suggestion
    "pooby's DMs",
```
Discord calls them Direct messages instead of Private messages